lib/govspeak.rb

```diff
@@ -118,6 +132,33 @@ def preprocess(source)
       source
     end
 
+    def legislative_list_footnote_definitions(source)
+      regex = /\$LegislativeList.*?\[\^\d\]*.*?\$EndLegislativeList/m
+      footnotes = source.scan(/\[\^(\d)\]:(.*)/)
+
+      if source.scan(regex).size.positive? && footnotes.size.positive?
+        list_items = footnotes.map do |footnote|
+          number = footnote[0]
+
```

Comment:
I'd be tempted to add a `text = footnote[1].strip` here, so that the way we're dealing with the `footnote` is consistent for both the text and number.